include/seqan3/alignment/configuration/align_config_result.hpp

```diff
@@ -96,7 +117,34 @@ template <typename with_type = detail::with_score_type>
 //!\endcond
 class result : public pipeable_config_element<result<with_type>, with_type>
 {
+    //!\brief The base type of this class.
+    using base_t = pipeable_config_element<result<with_type>, with_type>;
 public:
+    //!\brief The score type of the alignment result.
+    using score_type = score_t;
+
+    /*!\name Constructors, destructor and assignment
+     * \{
+     */
+    result() = default; //!< Defaulted.
+    result(result const &) = default; //!< Defaulted.
+    result & operator=(result const &) = default; //!< Defaulted.
+    result(result &&) = default; //!< Defaulted.
+    result & operator=(result &&) = default; //!< Defaulted.
+    ~result() = default; //!< Defaulted.
+
+    /*!\brief Construction from the result feature you want to compute (e.g. seqan3::with_score).
+     * \param[in] w The feature you want the alignment algorithm to compute (e.g. seqan3::with_score).
+     */
+    result(with_type w) : base_t{w} {}
+
+    /*!\brief Construction from the result feature you want to compute (e.g. seqan3::with_score).
+     * \param[in] w The feature you want the alignment algorithm to compute (e.g. seqan3::with_score).
+     * \param[in] s The score type to use (e.g. seqan3::using_score_type<int>).
+     */
+    result(with_type w, detail::score_type<score_t> SEQAN3_DOXYGEN_ONLY(s)) : base_t{w} {}
+    //!\}
+
     //!\privatesection
```

Comment:
Is this necessary for the code to work?

Reply:
I need the constructor from two parameters to be able to configure the score type.
The constructor with one parameter was formerly implicitly possible because it was an aggregate type, but since it isn't any more I also need a constructor for this. 
And a custom constructor automatically implies the rule of 6.